### DISCUSSION TITLE
Ability to stretch windows

### DISCUSSION THREAD

**Discussion by hasantahir** [godbout/Wooshy.docs#120] [Ideas]
Wondering if it's possible via Wooshy or Scrolla to stretch the window panels as shown in the video below:

https://user-images.githubusercontent.com/11236148/223448046-44568bf2-659b-4d29-903c-187ea712bd11.mov

The video is recorded in VS code and the Inspector doesn't catch anything.

Thanks, G!

**Comment by godbout** [comment]
yeah Wooshy ignores those because clicking on them is useless. what you would need to do is drag. hmm. not sure it belongs to Wooshy, nor Scrolla 🤔️ but yeah i've found i wanna do that sometimes also. need to process...

**Reply by hasantahir** [reply]
Yay! thanks 👀
